index.html

```diff
@@ -175,5 +175,67 @@ <h1 class="get-in-touch">
         </h1>
       </div>
     </section>
+    <section class="form-section">
+      <div class="form">
+        <form action="https://formspree.io/f/mknedrva" method="POST">
+          <div class="form-input">
+            <input
+              type="text"
+              name=""
+              id=""
+              placeholder="Full name"
```

Comment:
[ ] Make sure that The name field accepts up to 30 chars, not 50. it's in the project requirements. kindly fix that 🤓.